Brewfile.lock.json

```diff
@@ -0,0 +1,125 @@
+{
```

Comment:
Maybe let's include this file in a separate PR? In that PR I'd also add `swiftwasm/tap/carton` to `Brewfile`.